pkg/envoy/ads/stream.go

```diff
@@ -181,8 +185,9 @@ func respondToRequest(proxy *envoy.Proxy, discoveryRequest *xds_discovery.Discov
 	// Parse TypeURL of the request
 	typeURL, ok := envoy.ValidURI[discoveryRequest.TypeUrl]
 	if !ok {
-		log.Error().Msgf("Proxy %s: Unknown/Unsupported URI: %s",
-			proxy.String(), discoveryRequest.TypeUrl)
+		log.Error().Str(errcode.Kind, errcode.ErrInvalidURI.String()).
```

Comment:
s/ErrInvalidURI/ErrXDSInvalidTypeURI

Reply:
Changed errcode to `ErrInvalidXDSTypeURI`